cmd/onos-config/onos-config.go

```diff
@@ -78,59 +67,31 @@ func main() {
 
 	log.Info("Starting onos-config")
 
-	opts, err := certs.HandleCertPaths(*caPath, *keyPath, *certPath, true)
-	if err != nil {
-		log.Fatal(err)
-	}
-
-	atomixClient := atomix.NewClient(atomix.WithClientID(os.Getenv("POD_NAME")))
-
-	leadershipStore, err := leadership.NewAtomixStore(atomixClient)
-	if err != nil {
-		log.Fatal("Cannot load leadership atomix store ", err)
-	}
-
-	mastershipStore, err := mastership.NewAtomixStore(atomixClient, os.Getenv("POD_NAME"))
-	if err != nil {
-		log.Fatal("Cannot load mastership atomix store ", err)
-	}
-
-	deviceChangesStore, err := device.NewAtomixStore(atomixClient)
-	if err != nil {
-		log.Fatal("Cannot load device atomix store ", err)
-	}
-
-	networkChangesStore, err := network.NewAtomixStore(atomixClient)
-	if err != nil {
-		log.Fatal("Cannot load network atomix store ", err)
-	}
-
-	networkSnapshotStore, err := networksnap.NewAtomixStore(atomixClient)
+	modelRegistry, err := modelregistry.NewModelRegistry(modelregistry.Config{})
```

Comment:
we should be able to move this to manager as well correct?

Reply:
Can't move this unfortunately. Model registry must be initialized before the manager is created. The tests have a special implementation of this.

Reply:
hmm no. I think I can move it now. Let me try